client/src/client/testing/mod.rs

```diff
@@ -5,7 +5,7 @@ use futures::{Stream, StreamExt};
 use indexmap::IndexMap;
 
 use crate::{
-    api::{types::*, *},
+    api::{types::*, v1::SafeSelector, *},
```

Comment:
Reminder to self to introduce some reasonable major versioning scheme for the API.